src/test/java/org/apache/commons/io/FileUtilsDeleteDirectoryBaseTestCase.java

```diff
@@ -194,6 +194,29 @@ public void testDeleteParentSymlink2() throws Exception {
         assertEquals(1, randomDirectory.list().length, "Contents of sym link should not have been removed");
     }
 
+    @Test
+    public void testDeleteInvalidLinks() throws Exception {
+        File AFile = new File(top, "realParentDirA");
```

Comment:
These local variable names are unconventional, `AFile` -> `fileA` as local variable names should start with a lower case chaarcter.

Reply:
thnx, corrected.